spaceflights-pyspark/README.md

```diff
@@ -0,0 +1,44 @@
+# The `spaceflights-pyspark` Kedro starter
+
+## Overview
+
+This is a variation of the [spaceflights tutorial project](https://kedro.readthedocs.io/en/stable/tutorial/spaceflights_tutorial.html) described in the [online Kedro documentation](https://kedro.readthedocs.io) with `PySpark` setup.
+
+The code in this repository demonstrates best practice when working with Kedro and PySpark. It contains a Kedro starter template with some initial configuration and two example pipelines, and originates from the [Kedro documentation about how to work with PySpark](https://kedro.readthedocs.io/en/stable/tools_integration/pyspark.html).
+
+To use this starter, create a new Kedro project using the commands below. To make sure you have the required dependencies, run it in your virtual environment (see [our documentation about virtual environments](https://kedro.readthedocs.io/en/stable/get_started/prerequisites.html#virtual-environments) for guidance on how to get set up):
+
+```bash
+pip install kedro
+kedro new --starter=spaceflights-pyspark
+cd <my-project-name>  # change directory into newly created project directory
+```
```

Comment:
Do we want to keep this explanation or remove this from the new starters? @amandakys